Sources/NIOCore/SystemCallHelpers.swift

```diff
@@ -121,12 +128,24 @@ enum SystemCalls {
         }
     }
 
+#if os(Windows)
+    @inline(never)
+    static func open(file filename: UnsafePointer<CChar>, oFlag oflag: CInt,
+                     mode pmode: CInt) throws -> CInt {
```

Comment:
This doesn't need a new signature, just a new implementation. Can we only replace the implementation?